core-build/gulp-core-build/src/tasks/jest.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "title": "Jest Task configuration",
+  "description": "Defines Jest task configuration. Same definitions as Jest CLI has.",
+
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "isEnabled": {
+      "description": "Indicate whether the test is enabled",
+      "type": "boolean",
+      "default": false
```

Comment:
>  "default": false [](start = 4, length = 18)

I don't believe this is a supported feature in the JSON Schema.

Reply:
I'm remembering that we disabled metaschema validation (i.e. checking that your schema is correctly formatted) because of a Z-Schema bug:

[https://github.com/zaggino/z-schema/issues/191#issuecomment-325522975](https://github.com/zaggino/z-schema/issues/191#issuecomment-325522975)

We should probably fix this.  It's not difficult.

---
In reply to: [146365180](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r146365180) [](ancestors = 146365180)

Reply:
I opened VSO 463299

---
In reply to: [146367602](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r146367602) [](ancestors = 146367602,146365180)